### PR TITLE
Add 404 Page

### PR DESCRIPTION
**Pull request by roshni73**
## Addresses:
- https://github.com/IFRCGo/go-web-app/issues/1358

## Changes
- Add 404 page 

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: app/src/views/FourHundredFour/styles.module.css [hunk 1]
```diff
@@ -2,10 +2,12 @@
     display: flex;
     flex-direction: column;
 
+
     .main-section-container {
         display: flex;
         flex-direction: column;
         flex-grow: 1;
+        background-image: url('./background.svg');
```
Comment (tnagorra) [inline app/src/views/FourHundredFour/styles.module.css]: We should make sure that build-tool / linter can identify if the image exists.

File: app/src/views/FourHundredFour/styles.module.css [hunk 2]
```diff
@@ -14,32 +16,41 @@
...
+                background: repeating-linear-gradient(
+                    -50deg,
+                    red 10px,
+                    transparent ,
+                    transparent 20px
+                );
+                -webkit-background-clip: text;
+                -moz-background-clip: text;
+                -webkit-text-fill-color: transparent;
+                font-size: 8rem;
```
Comment (tnagorra) [inline app/src/views/FourHundredFour/styles.module.css]: Don't we have a design token for this?

File: app/src/views/FourHundredFour/styles.module.css [hunk 3]
```diff
@@ -14,32 +16,41 @@
...
+                font-weight: bold;
```
Comment (tnagorra) [inline app/src/views/FourHundredFour/styles.module.css]: Don't we have a design token for this?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/src/views/FourHundredFour/styles.module.css (hunk 2 above)
Comment (frozenhelium) [reply app/src/views/FourHundredFour/styles.module.css]: We should not use browser suffixes

File: app/src/views/FourHundredFour/styles.module.css (hunk 1 above)
Comment (frozenhelium) [reply app/src/views/FourHundredFour/styles.module.css]: We can import this image and use as background layer in the `index.tsx`

**Review by frozenhelium — CHANGES_REQUESTED** [review]

File: app/src/views/FourHundredFour/background.svg
Comment (frozenhelium) [inline app/src/views/FourHundredFour/background.svg:1]: This should be moved to assets and renamed properly

File: app/src/views/FourHundredFour/styles.module.css (hunk 2 above)
Comment (frozenhelium) [reply app/src/views/FourHundredFour/styles.module.css:39]: We can simply use the image for 404 from figma

**Review by frozenhelium — CHANGES_REQUESTED** [review]
Also the background is too prominent at the moment, we should mute it.

File: app/src/views/FourHundredFour/styles.module.css (hunk 3 above)
Comment (frozenhelium) [reply app/src/views/FourHundredFour/styles.module.css]: We should also define the design tokens for the gradients that's being used here

**Review by shreeyash07 — APPROVED** [review]

File: app/src/views/FourHundredFour/i18n.json [hunk 4]
```diff
@@ -2,11 +2,9 @@
...
         "uhohPageNotFoundTitle":"IFRC GO - Page not found",
-        "uhohPageNotFound":"Page not found",
-        "uhohPageDescription":"The requested page does not exist or may have been removed.",
-        "uhohAreYouSureUrlIsCorrect":"Are you sure the URL is correct?",
-        "uhohGetInTouch":"Get in touch",
-        "uhohWithThePlatformTeam":"with the platform team.",
-        "uhohExploreOurHomepage":"Explore our homepage"
+        "uhohOops":"Oops!",
+        "uhohPageDescription1": "This page seems to have wandered off.",
+        "uhohPageDescription2":"let's go back home and try again.",
```
Comment (samshara) [inline app/src/views/FourHundredFour/i18n.json]:
```suggestion
        "uhohPageDescription2":"Let's go back home and try again.",
```

File: app/src/views/FourHundredFour/index.tsx [hunk 5]
```diff
@@ -18,43 +18,38 @@ export function Component() {
             title={strings.uhohPageNotFoundTitle}
             mainSectionContainerClassName={styles.mainSectionContainer}
             mainSectionClassName={styles.main}
+            beforeHeaderContent={(
+                <img
+                    className={styles.backgroundLayer}
+                    src={backgroundImage}
+                    alt=""
```
Comment (samshara) [inline app/src/views/FourHundredFour/index.tsx]: It's always good to add alt text. We could use a generic placeholder.

**Review by samshara — CHANGES_REQUESTED** [review]
Seems like the page styling broken.

**Review by samshara — APPROVED** [review]

**Review by samshara — APPROVED** [review]

**Pull request merged**